Makefile

```diff
@@ -32,6 +31,8 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 
 test :
+    python ./setup.py develop
+    py.test
```

Comment:
I think that I'll remove the setup step from the test target in the Makefiles. Otherwise it wouldn't be possible to test an officially installed version.